_data/axeljovanny/axeljovanny.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Axel Jovanny Quintero
+institution: Universidad Tecnológica del Norte de Guanajuato
+profile_pic: axel.png
+quote: "It is easier to believe in your community than to believe that you are alone, keep calm and Fierro Pariente."
```

Comment:
Please follow the given [instruction](https://github.com/education/graduation#3-add-your-profile-information) by removing the quotes to maintain the same format. With that being said, your character quote count exceeds the given limit-100, please modify it!